ssh/tailssh/tailssh.go

```diff
@@ -165,6 +167,13 @@ func (srv *server) handleSSH(s ssh.Session) {
 		logf("ssh: user Lookup %q: %v", localUser, err)
 		s.Exit(1)
 	}
+	ci.localUser = localUser
+	uid, err := strconv.ParseUint(lu.Uid, 10, 32)
+	if err != nil {
+		logf("ssh: user Lookup %q: %v", localUser, err)
+		s.Exit(1)
```

Comment:
don't forget "return" after this

Reply:
Ah, the line above is also borked. Will fix